### DISCUSSION TITLE
How do I change the language.

### DISCUSSION THREAD

**Discussion by Creed-22** [barry-ran/QtScrcpy#849] [Q&A] [answered]
How do I change the language to English? Currently it’s in Chinese

**Comment by kaheichanturtle** [comment, marked as answer]
Hey! not sure if you are still asking the question, but I just installed it today, and used it, got the same problem and searched google. your's the first result that popped up on google, and I'm disappointed no one replied. I as a developer then dug into the app files and found how to fix using this easy method: 

1: right click on the app
2: click show package contents
3: click contents, then open the "info.plist" file. 
4: in the <string> part, it should be zh_chinese or something similar. Change it to Us-EN 
5: there you go bud! have fun

tell me if you have any more difficulties

**Reply by Creed-22** [reply]
Damn it’s almost a year since I asked this question! And I don’t even use this software anymore. But anyway thanks for the solution mate. Someone who might have the same problem will find your solution helpful. Cheers 👍

**Reply by NicksonYap** [reply]
change `zh-Hans` to `en-US`

**Comment by zyahav** [comment]
Thank you so much for your answer. It helped me. I appreciate it very much. 


------------------------
You said:
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>CFBundleDevelopmentRegion</key>
	<string>zh-Hans</string>
	<key>CFBundleExecutable</key>
	<string>QtScrcpy</string>
	<key>CFBundleGetInfoString</key>
	<string>Created by rankun</string>
	<key>CFBundleIconFile</key>
	<string>QtScrcpy</string>
	<key>CFBundleIdentifier</key>
	<string>rankun.QtScrcpy</string>
	<key>CFBundleInfoDictionaryVersion</key>
	<string>6.0</string>
	<key>CFBundleName</key>
	<string>QtScrcpy</string>
	<key>CFBundlePackageType</key>
	<string>APPL</string>
	<key>CFBundleShortVersionString</key>
	<string>3.3.3</string>
	<key>CFBundleSupportedPlatforms</key>
	<array>
		<string>MacOSX</string>
	</array>
	<key>CFBundleVersion</key>
	<string>3.3.3</string>
	<key>LSMinimumSystemVersion</key>
	<string>10.10</string>
	<key>NSAppleEventsUsageDescription</key>
	<string></string>
	<key>NSHumanReadableCopyright</key>
	<string>Copyright © 2018-2038 rankun. All rights reserved.</string>
	<key>NSMainStoryboardFile</key>
	<string>Main</string>
	<key>NSPrincipalClass</key>
	<string>NSApplication</string>
	<key>NSSupportsAutomaticGraphicsSwitching</key>
	<true/>
</dict>
</plist>

**Comment by ricafolio** [comment]
new method for latest version (v3.3.3) in mac:

1. Go to **Finder**, head over to menu bar
2. Select **Go** > **Applications**
3. Right click **QtScrcpy** > click **Show package contents**
4. Navigate to folder `Contents` > `MacOS` > `config`
5. Open `config.ini` in your editor and change entry to `Language=en_US`
